ibm/service/kubernetes/resource_ibm_container_dedicated_host.go

```diff
@@ -0,0 +1,482 @@
+// Copyright IBM Corp. 2017, 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	v2 "github.com/IBM-Cloud/bluemix-go/api/container/containerv2"
+	"github.com/IBM-Cloud/bluemix-go/bmxerror"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+)
+
+const (
+	created       = "created"
+	creating      = "creating"
+	createFailed  = "create_failed"
+	createPending = "create_pending"
+	deleted       = "deleted"
+	deleting      = "deleting"
+	deleteFailed  = "delete_failed"
+
+	DedicatedHostStateCreatePending = createPending
+	DedicatedHostStateCreating      = creating
+	DedicatedHostStateCreated       = created
+	DedicatedHostStateCreateFailed  = createFailed
+	DedicatedHostStateDeleting      = deleting
+	DedicatedHostStateDeleted       = deleted
+	DedicatedHostStateDeleteFailed  = deleteFailed
+
+	DedicatedHostCreateTimeout = time.Minute * 30
```

Comment:
instead of using DedicatedHostCreateTimeout please use the Terraform timeouts which provides flexbility for end user to  repalce the value
https://www.terraform.io/plugin/sdkv2/resources/retries-and-customizable-timeouts (use respective timeouts in each function of create,update,delete)

Reply:
done